.github/workflows/check_smoke_tests.yml

```diff
@@ -0,0 +1,112 @@
+name: check smoke tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  smoke_tests:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out repository
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Start PostgreSQL Database
```

Comment:
@cbrinson-rise8 [services](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idservices) make dependencies like this a bit easier to manage.  For instance, take a look at this [postgres service we create for unit tests](https://github.com/CDCgov/RecordLinker/blob/main/.github/workflows/check_unit_tests.yml#L17-L23).  We can then use that service in any of our steps by [referencing the hostname that matches the service name](https://github.com/CDCgov/RecordLinker/blob/main/.github/workflows/check_unit_tests.yml#L73) (in this case, "postgres").